spec/storagefreak_spec.coffee

```diff
@@ -425,6 +425,42 @@ describe 'StorageFreak', ->
       @instance.search('as')
       expect(spy).to.not.be.called
 
+    it 'matches only infix data by default', (done)->
+      @instance._data = {
+        add: 'addas'
```

Comment:
You could change `add: 'addas`' to `addasadd: 'addasadd'` to make it even clearer that we're operating in `infix` mode. (We could keep `addas` for `suffix` mode).